bazel/repository_locations.bzl

```diff
@@ -37,6 +37,11 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "xxHash-0.6.5",
         urls = ["https://github.com/Cyan4973/xxHash/archive/v0.6.5.tar.gz"],
     ),
+    com_github_rshriram_sqlparser = dict(
+        sha256 = "2869642cb868b26e242cb8ff1c196634171b1a826f797780fddbf90d7414a979",
+        strip_prefix = "sql-parser-6197eecaab3a9f50f48dd253e0c5ef3d0c78add0",
+        urls = ["https://github.com/rshriram/sql-parser/archive/6197eecaab3a9f50f48dd253e0c5ef3d0c78add0.tar.gz"],
```

Comment:
Is this where this is going to live forever? Can you host this in a vmware GH org? Or do you want an EnvoyProxy org repo for this?

Reply:
I dont mind hosting this in Envoyproxy org. Getting a VMware GH org is going to take time as they want to only host some public projects (not temporary forks). Also, this is a temporary state of things until this parser eventually moves into Envoy codebase.

Reply:
So what is the long term plan here? Can you describe in more detail and also talk about timelines?

Reply:
This parser is derived from another project (hyrise/sql-parser) and heavily modified. The parser needs to be cleaned up and bazel-fied, not to mention a lot of code changes to meet Envoy's formatting guidelines. There is some bazel vodoo involved in running flex/yacc as well which I am not aware of. Finally, the parser generates C code FYI. No control over that and no control over the naming, types, etc. 

timeline wise, it would take a couple of months until this parser code moves into Envoy as a SQL parser library. Hosting it in the envoyproxy org in the interim is fine by me.

Reply:
OK I see. IMO it probably makes sense to keep the parser as a separate library anyway. I don't have a strong opinion here but happy to setup a repo in EnvoyProxy org for you if that would work best (are we forking from somewhere else)?

Reply:
its forked from https://github.com/hyrise/sql-parser which is a generic SQL92 parser. We added stuff to the parser for mysql syntax parsing. Will add more over time for postgres, etc.

Reply:
Do you intend to upstream the parser changes? Or unlikely? If unlikely, do you want me to fork sql-parser into EnvoyProxy org and you can make the changes there? If possible can you perhaps hop on the community call tomorrow and we can discuss?